packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.component.js

```diff
@@ -154,7 +179,7 @@ export default class MultiSelectTag extends React.Component {
 		this.setState(oldState => {
 			const currentValue = value === undefined ? oldState.value : value;
 			const currentProps = props === undefined ? this.props : props;
-			let suggestions = currentProps.schema.titleMap
+			let suggestions = this.getTitleMap(currentProps)
```

Comment:
Not a big fan of let, but you already know that :)

Reply:
Yes but the use here is totally right.
It's still scoped within the block, and no fancy things within those 10 LOC.